setup.py

```diff
@@ -521,7 +571,12 @@ def _read_requirements(filename: str) -> List[str]:
 }
 if envs.VLLM_USE_PRECOMPILED:
     ext_modules = []
-    package_data["vllm"].append("*.so")
+    wheel_location = os.getenv(
+        "VLLM_PRECOMPILED_WHEEL_LOCATION",
+        "https://vllm-wheels.s3.us-west-2.amazonaws.com/nightly/vllm-1.0.0.dev-cp38-abi3-manylinux1_x86_64.whl"
```

Comment:
will this wheel be stable? I found that python 3.8 suppoort is dropped. #10038

Reply:
this wheel works for python 3.8+ , not just python 3.8 . we will keep the wheel name.